core/src/main/javascript/reportviewer/reportviewer.js

```diff
@@ -1242,13 +1242,13 @@ define([ 'common-ui/util/util', 'common-ui/util/timeutil', 'common-ui/util/forma
                 case "QUEUED":
                 case "WORKING":
                   // Although we are hiding the screen latter, update the label anyway;
-                  $('#notification-message').html(_Messages.getString('LoadingPage') + " " + mainJobStatus.page + " " + _Messages.getString('Of') + " " + mainJobStatus.totalPages);
+                  $('#notification-message').html(xssUtil.sanitizeHtml(_Messages.getString('LoadingPage') + " " + mainJobStatus.page + " " + _Messages.getString('Of') + " " + mainJobStatus.totalPages));
```

Comment:
Idem, for use of `xssUtil.setHtml(...)` given a jQuery object.

Reply:
Updated